extensions/amp-lightbox-viewer/0.1/amp-lightbox-viewer.js

```diff
@@ -295,6 +335,13 @@ export class AmpLightboxViewer extends AMP.BaseElement {
   setupElement_(element) {
     this.updateStackingContext_(element, /* reset */ false);
     element.classList.add('amp-lightboxed');
+    // update description box
+    const descText = this.manager_.getDescription(element);
```

Comment:
One thing I don't have here is: after use close the lightbox viewer. The description box viewability state won't be reset. Which means if they disable description box and close lightbox viewer, it will still be hidden when user open lightbox viewer later. 
Which one you think is more user friendly? reset the state on close or not? @aghassemi